terraform/files/bin/bootstrap.sh

```diff
@@ -50,6 +50,8 @@ else
   sudo mv kustomize /usr/local/bin/
 fi
 
+install_kube_ps1.sh
+
```

Comment:
Minor: Would be great (sooner is better) to introduce the Release-Notes-R6.md doc and fill it appropriately.